frontend/src/components/Forms/FormLayout/index.module.css

```diff
@@ -6,11 +6,21 @@
 }
 
 .headerWrapper {
-  margin-bottom: 16px;
+  margin-bottom: 12px;
   text-align: center;
 }
 
 .subtitleWrapper {
-  margin-bottom: 72px;
+  margin-bottom: 56px;
   text-align: center;
 }
+
+.subtitleInterests {
+  max-width: 200px;
+  margin-bottom: 40px;
+}
+
+.subtitleOtherInterest {
+  max-width: 200px;
+  margin-bottom: 80px;
+}
```

Comment:
What design are you basing the `margin-bottom` values on?

Reply:
Thank you! By the way, you can link to frames in Figma.

<img width="712" alt="Screenshot 2022-08-23 at 15 43 13" src="https://user-images.githubusercontent.com/26620750/186173957-d2e82da6-3f61-4e18-a9f4-32cf857b0590.png">

Reply:
Oh, didn't know that :sweat_smile: Thanks, will do next time